api/lib/session/bloc/session_bloc.dart

```diff
@@ -49,7 +49,7 @@ class SessionBloc extends BroadcastBloc<SessionEvent, SessionState> {
   void _onMessageSent(OnMessageSent event, Emitter<SessionState> emit) {
     emit(
       state.copyWith(
-        messages: [...state.messages, event.chat],
+        messages: [event.chat,...state.messages],
```

Comment:
What's the reason for this change?

Reply:
did it coz in chat component need to show list in reverse order

Reply:
We should not change the order of the chat on backend, it's the source of truth. Manipulate the chat on frontend